### DISCUSSION TITLE
GitHub Campus Expert Rejection: Feedback Please! 🙏

### DISCUSSION THREAD

**Discussion by talha1230** [community/community#113331] [GitHub Education] [answered]
### Hi everyone,

Just got the news that my GitHub Campus Expert application didn't make the cut. Bummer, right? 🤷‍♂️
Now, I ain't gonna lie, it stung a bit. I mean, I've been grinding away at my coding projects, burning the midnight oil like a true bachelor nerd. So, when GitHub said, "**Sorry, not this time**," I couldn't help but feel a little deflated.
So, here I am, reaching out to you fine folks for some feedback. I wanna know where I went wrong, what I could've done better, and how I can up my game for next time.

Thanks a bunch, and keep coding on!

**Comment by samus-aran** [comment, marked as answer]
Hi @talha1230 

While we understand your desire to gain insight into the selection process, we regret to inform you that due to the high volume of applications we received, we are unable to provide specific individualized feedback to all applicants.

However, we want to offer some of the things we looked for that can help you apply for the next Campus Experts cohort:

- **Teamwork:** evidence of the understanding that it takes more than one person to build an event. "I created an event for N people at my campus" vs. "I was part of the team responsible for IT support during our hackathon, but I also jumped in to help my colleague with picking up food and ensuring that we had our bills paid." 

- **Great evidence of community building already happening:** links to photos, websites, or more that show events or community work happening. 

- **Chat GPT free:** Be careful not to overuse ChatGPT and other generative AI that can lead your application to sound generic and lackluster. Aim to make your application stand out and unique to demonstrate your individuality. We can tell when it's mostly AI writing the answer.
Using generative AI to help with spelling, misused idioms, and general clarity is ok, but we would rather see the applicants write their own original writing explaining what they've been working on in the community rather than depending on a completely generated response. 

These suggestions are not about any one application in particular, but these are very common characteristics of applications we have decided to pass on. We truly value your interest and participation in the application process, and we hope that you will continue to engage with us and explore other avenues for personal and professional development on GitHub.
